src/main/java/hek/models/EventType.java

```diff
@@ -5,8 +5,12 @@
 
 /**
  * Created by ahmetkucuk on 01/10/15.
+ * @author jookimmy
  */
 public enum EventType {
+    /**
+     * Acronyms representing all the different types of solar events.
```

Comment:
Please either add long name for each event type or add this url here: http://www.lmsal.com/hek/VOEvent_Spec.html